src/Cli/dotnet/BuiltInCommandsCatalog.cs

```diff
@@ -49,6 +49,12 @@ public static class BuiltInCommandsCatalog
                 // aka.ms target: https://docs.microsoft.com/dotnet/articles/core/tools/dotnet-clean
                 DocLink = "https://aka.ms/dotnet-clean"
             },
+            ["cleanup"] = new BuiltInCommandMetadata
+            {
+                Command = CleanupCommand.Run,
+                // aka.ms target: https://docs.microsoft.com/dotnet/articles/core/tools/dotnet-cleanup
```

Comment:
@jmarolf Currently this aka.ms link routes to https://docs.microsoft.com/en-us/dotnet/core/tools/elevated-access?tabs=windows

Reply:
[This](https://docs.microsoft.com/dotnet/articles/core/tools/dotnet-cleanup) now 404s (which is believe is the correct behavior for now). Adding the documentation in [this PR](https://github.com/dotnet/docs/pull/25082)